skbio/stats/composition.py

```diff
@@ -644,14 +646,15 @@ def ancom(table, grouping,
           tau=0.02,
           theta=0.1,
           multiple_comparisons_correction=None,
-          significance_test=None):
+          significance_test=None,
+          percentiles=[0.0, 25.0, 50.0, 75.0, 100.0]):
```

Comment:
lists are mutable and shouldn't be used as default values in Python. Can you change this to `None` or a tuple?